fast-list.js

```diff
@@ -567,8 +567,7 @@ function computeIndices(source, geometry) {
   }
   if (endIndex > lastIndex) {
     extra = endIndex - lastIndex;
-    endIndex = lastIndex;
-    endIndex = Math.max(0, startIndex + extra);
```

Comment:
I think this was supposed to be `startIndex`
```
endIndex = lastIndex;
startIndex = Math.max(0, startIndex - extra);
```

if start index is smaller than 0
  - we set it to 0
  - we add the extra to the end index (making the window bigger)

if endIndex bigger than the last index
  - we set it to the last index
  - we remove the extra to the _start_ index (making the window bigger)